annotations/update_pending_annotations.py

```diff
@@ -0,0 +1,152 @@
+import synapseclient
+from synapseclient import Table, RowSet
+import argparse
+import pandas as pd
+from attribute_dictionary import PUBLICATION_DICT
+
+
+### Login to Synapse ###
+def login():
+
+    syn = synapseclient.Synapse()
+    syn.login()
+
+    return syn
+
+
+def get_args():
+
+    parser = argparse.ArgumentParser(
+        description=
+        'Get synapse table id of annotations to be editd and synapse table id of controlled vocabulary mappings'
+    )
+    parser.add_argument(
+        '-t',
+        '--table_id',
+        type=str,
+        default='syn21868591',
+        help=
+        'Synapse table id where annotations will be updated. Probably the Publications Merged table.'
+    )
+    parser.add_argument('-m',
+                        '--manifest_path',
+                        type=str,
+                        help='Path where updated manifest is stored.')
+    return parser.parse_args()
+
+
+def get_updated_df(manifest):
+
+    updated_df = pd.read_csv(manifest, index_col=False).fillna("")
```

Comment:
I think this line of code is simple/small enough that you don't need to put this into a separate function.